docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -97,25 +97,6 @@ function sitenow_query_administerusersbyrole_edit_access_alter(AlterableInterfac
   }
 }
 
-/**
```

Comment:
Unless this is getting moved somewhere else I think we still need this

Reply:
Yeah, nope--definitely still need it, and it wasn't getting moved. Committed changes to an old version of sitenow.profile. Got it updated.